com.unity.cinemachine/Documentation~/CinemachineVirtualCameraAim.md

```diff
@@ -1,13 +1,13 @@
 # Rotation Control
 
-Use the Rotation Control behaviours to specify how to rotate the CmCamera. To change the camera’s position, use a [Position Control](CinemachineVirtualCameraBody.md) behaviour.
+Use the Rotation Control behaviors to specify how to rotate the CmCamera. To change the camera’s position, use a [Position Control](CinemachineVirtualCameraBody.md) behaviour.
 
 ![Aim, with the Composer behaviour (red)](images/CinemachineAim.png)
 
 
-Cinemachine includes these behaviours for rotating a CmCamera:
+Cinemachine includes these behaviors for rotating a CmCamera:
 
-- [__None__](CinemachineAimDoNothing.md): Do not procedurally rotate the CmCamera.  The rotation can be controlled by object parenting, or by custom script, or by animation.
+- [__None__](CinemachineAimDoNothing.md): Do not procedurally rotate the CmCamera. The rotation can be controlled by object parenting, a custom script, or by animation.
```

Comment:
This feels inconsistent.  I think you should leave the middle "by" or take out the last one.